src/ResourceManager/Reservations/ChangeLog.md

```diff
@@ -20,6 +20,10 @@
 -->
 ## Current Release
 
+## Version 0.1.2
```

Comment:
@corquiri you can remove the version header, the text content you're adding should go under the `## Current Release` header